assets/dev/js/editor/components/saver/behaviors/footer-saver.js

```diff
@@ -20,99 +30,39 @@ module.exports = class FooterSaver extends Marionette.Behavior {
 		};
 	}
 
-	initialize() {
-		/**
-		 * @type {Component}
-		 */
-		this.saver = $e.components.get( 'document/save' );
+	initialize( options ) {
+		this.document = options.document || elementor.documents.getCurrent();
 
-		this.saver
-			.on( 'before:save', this.onBeforeSave.bind( this ) )
-			.on( 'after:save', this.onAfterSave.bind( this ) )
-			.on( 'after:saveError', this.onAfterSaveError.bind( this ) )
-			.on( 'page:status:change', this.onPageStatusChange );
+		elementor.footerSaver = this;
 
-		elementor.on( 'document:loaded', () => {
-			elementor.settings.page.model.on( 'change', this.onPageSettingsChange.bind( this ) );
-		} );
+		if ( ! FooterSaver.initOnce ) {
+			FooterSaver.initHooks();
 
-		elementor.channels.editor.on( 'status:change', this.activateSaveButtons.bind( this ) );
+			FooterSaver.initOnce = true;
+		}
 	}
 
-	activateSaveButtons( hasChanges ) {
-		hasChanges = hasChanges || 'draft' === elementor.settings.page.model.get( 'post_status' );
+	activateSaveButtons( status ) {
+		const hasChanges = status || 'draft' === elementor.settings.page.model.get( 'post_status' );
 
 		this.ui.buttonPublish.add( this.ui.menuSaveDraft ).toggleClass( 'elementor-disabled', ! hasChanges );
 		this.ui.buttonSaveOptions.toggleClass( 'elementor-disabled', ! hasChanges );
 	}
 
 	onRender() {
 		this.setMenuItems( elementor.settings.page.model.get( 'post_status' ) );
-		this.setLastEdited( elementor.config.document.last_edited );
+		// TODO: Check with mati.
```

Comment:
Add a `last_edited` entry to wp-content/plugins/elementor/tests/qunit/mock/documents/document-1.json